z3/tests/ops.rs

```diff
@@ -209,7 +209,7 @@ fn test_ast_children() {
     assert_eq!(not_a.nth_child(1), None);
 
     let b = Bool::new_const(&ctx, "b");
-    let a_or_b = Bool::or(&ctx, &[&a, &b]);
+    let a_or_b = Bool::or(&ctx, &[a.clone(), b.clone()]);
```

Comment:
Could you add a comment here that explains the usage of `.clone()` to make sure that passing values rather than references works?

Reply:
Sure